authz-model/src/main/java/com/symphony/s2/authz/canon/facade/EntitlementMappingInfo.java

```diff
@@ -0,0 +1,103 @@
+/**
+ * Copyright 2020 Symphony Communication Services, LLC.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
```

Comment:
If we don't need to put anything in the facade remove the facade true label in the model, delete this file and regenerate. The package name will change so you will have to fix up that.